indigo_app/static/javascript/indigo/models.js

```diff
@@ -61,10 +61,12 @@
       // originally triggered by a content change
       if (options && options.fromContent) return;
 
-      // rewrite all eIds before setting the content
-      new indigoAkn.EidRewriter().rewriteAllEids(this.xmlDocument.documentElement);
-      // rewrite all attachment FRBR URI work components too
-      new indigoAkn.WorkComponentRewriter().rewriteAllAttachmentWorkComponents(this.xmlDocument.documentElement);
+      if (Indigo.Preloads.provisionEid === "") {
```

Comment:
:( always needs to be done, in case you quick-edit something and give it a duplicate eid